### PR TITLE
Add travis-ci

### PR DESCRIPTION
**Pull request by HugoGresse**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -21,6 +21,9 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    lintOptions {
+        abortOnError false
```
Comment (HugoGresse) [inline app/build.gradle]: I'll not do that :D

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  components:
+  - tools
+  - tools # see https://github.com/travis-ci/travis-ci/issues/6040#issuecomment-219367943)
+  - platform-tools
+  - build-tools-$ANDROID_BUILD_TOOLS
+  - android-$ANDROID_API
+  - android-$EMULATOR_API_LEVEL
+  - extra-google-m2repository
+  - extra-android-m2repository
+  - sys-img-armeabi-v7a-addon-google_apis-google-$ANDROID_API_LEVEL
```
Comment (HugoGresse) [inline .travis.yml]: we do not need emulator if we only do plain java UT

Comment (HugoGresse) [reply .travis.yml]: BTW, emulator on travis is very not stable IMO

Comment (Viewtiful) [reply .travis.yml]: Just wanted to try it but yes you are right

**Comment by HugoGresse** [comment]
LGTM

**Pull request merged**